doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md

```diff
@@ -1,23 +1,23 @@
-## Procedural documentation for DSFinV-K generation
+# Procedural documentation for DSFinV-K generation
 
 
-### DSFinV-K export
+## DSFinV-K export
 
 You can download the current version of the DSFinV-K specification [here](https://www.bzst.de/DE/Unternehmen/Aussenpruefungen/DigitaleSchnittstelleFinV/digitaleschnittstellefinv_node.html).
 
-Based on the version 2.2 of the DSFinV-K specification, this chapter explains how the DSFinV-K export is structured, shows how the previously described input values are mapped by fisklatrust to the files and data of the DSFinV-K export and defines how additional, for the DSFInV-K required, values can be sent to the ft.Middleware. Furthermore, it describes how the marking of actions (DE: Vorgänge) can be made by connecting business actions (DE: Geschäftsvorfälle) and other procedures, occurrences and events (DE: Andere Vorgänge). 
+Based on the version 2.2 of the DSFinV-K specification, this chapter explains how the DSFinV-K export is structured, how the previously described input values are mapped by fiskaltrust to the files and data of the DSFinV-K export, and defines how additional, for the DSFinV-K required values can be sent to the fiskaltrust.Middleware. Furthermore, it describes how the marking of actions (DE: Vorgänge) can be made by connecting business actions (DE: Geschäftsvorfälle) and other procedures, occurrences and events (DE: Andere Vorgänge).  
 
-#### Structure
+### Structure
 
 The DSFinV-K export is divided into the following sections/modules: 
 
 - Single recordings module (DE: Einzelaufzeichnungsmodul)
 - Master data module (DE: Stammdatenmodul)
 - Cashpoint closing module (DE: Kassenabschlussmodul)
 
-Each module consists of several files. In the following we will take a look at the individual modules especially the content of the files and data. Details about the meaning of the files and their individual fields can be found in the official DSFinV-K specification.
+Each module consists of several files. In the following, we will look at the individual modules, especially the files and data content. Details about the meaning of the files and their individual fields can be found in the official DSFinV-K specification.
```

Comment:
In the following section, we will look at some (are we looking at all of them?) individual modules, especially the files and data content ones.